desc/compute/_field.py

```diff
@@ -27,7 +27,7 @@
 
 
 @register_compute_fun(
-    name="B0",
+    name="psi_r/sqrt(g)",
```

Comment:
I know we don't normally like to rename existing variables, but I will argue that in this case it is desirable. The old quantity `"B0"` was only ever used internally for computing $B^\zeta$ and is very unlikely to ever be computed by the user directly. `"B0"` is now the magnetic field strength on axis, which is what this variable is commonly used for in the literature.